tests/test_build_featurizer.py

```diff
@@ -228,12 +227,12 @@ def test_initialize_model_wrong_weights():
 
 
 INITIALIZE_MODEL_CASES = [
-    ('squeezenet', 67, (1, 227, 227, 3)),
-    ('vgg16', 23, (1, 224, 224, 3)),
-    ('vgg19', 26, (1, 224, 224, 3)),
-    ('resnet50', 176, (1, 224, 224, 3)),
-    ('inceptionv3', 313, (1, 299, 299, 3)),
-    ('xception', 134, (1, 299, 299, 3)),
+    ('squeezenet', [67], (1, 227, 227, 3)),
```

Comment:
Added because layers can change across versions of keras. Want to be able to check that expected layers is in a list of expected layers.